IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.m

```diff
@@ -54,6 +54,8 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 {
     MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Beginning interactive broker extension flow.");
     
+    [self addQRPINParamIfNeeded];
```

Comment:
Actually, I didn't see this being updated for other scenarios, like legacy broker flow and non-broker flow. Should it be updated there too?